.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/cyril.ruffa/Documents/Symphony/symphony-bdk-ui-toolkit-components/src/index.ts":"1"},{"size":48,"mtime":1639072644512}]
```

Comment:
Looks like it was pushed even though it is in your .gitignore

Reply:
yes i ignore why i have removed it manually